pom.xml

```diff
@@ -330,7 +330,7 @@
                 <groupId>org.wso2.am</groupId>
                 <artifactId>org.wso2.am.integration.common.test.utils</artifactId>
                 <!-- TODO:Dinusha - bump the version to 4.0.0-latest -->
-                <version>4.0.0-beta</version>
+                <version>4.0.0-SNAPSHOT</version>
```

Comment:
Lets move this to root pom and refer from there. Lets not keep the version in child pom files